adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -425,6 +427,20 @@ func createRoute(params *routeCreateParams) *routev3.Route {
 		}
 	}
 
+	conf, errReadConfig := config.ReadConfigs()
+	if errReadConfig != nil {
+		logger.LoggerOasparser.Fatal("Error loading configuration. ", errReadConfig)
+	}
+
+	if !conf.Security.Adapter.EnableOutboundAuthHeader {
+		logger.LoggerOasparser.Debugf("removeHeader: %v", authHeader)
+		if authHeader == "" {
+			authHeader = conf.Security.Adapter.AuthorizationHeader
+		}
+		removeHeaders = append(removeHeaders, authHeader)
+		removeHeaders = append(removeHeaders, "Internal-Key")
```

Comment:
shall we introduce a constant here?

Reply:
Fixed with https://github.com/wso2/product-microgateway/pull/1785/commits/5e7db9dba2577979211d7713f6690403ca650326